src/BloomBrowserUI/bookEdit/js/bloomEditing.js

```diff
@@ -489,6 +490,16 @@ function MakeSourceTextDivForGroup(group) {
   });
 }
 
+function checkMightCauseHorizontallyOverlappingBubbles(element) {
```

Comment:
Would it be appropriate to investigate whether the window is adequately wider than the page? I have vague recollections of bubbles doing weird things if there isn't much room for them.
That might appropriately be thought of as a separate fix if we decide it's useful, of course...just asking.